ct-app/core/model/peer.py

```diff
@@ -100,14 +102,30 @@ def airdrop_reward(self):
         if self.economic_model is None:
             raise ValueError("Economic model not set")
 
-        return self.expected_reward * (1 - self.economic_model.budget.s)
+        expected_reward = (
+            self.apr_percentage
+            / 100.0
+            * self.split_stake
+            * self.economic_model.budget.period
+            / (60 * 60 * 24 * 365)
+        )
+
+        return expected_reward * (1 - self.economic_model.budget.s)
```

Comment:
Code repetition is the godmother of all bugs, please, refactor that as a single private method.